cmake/templates/OpenVINOConfig.cmake.in

```diff
@@ -224,6 +224,6 @@ if(_need_package_name_reset)
     unset(_need_package_name_reset)
 endif()
 
-unset(${CMAKE_FIND_PACKAGE_NAME}_PaddlePaddle_FOUND)
+unset(${CMAKE_FIND_PACKAGE_NAME}_paddle_FOUND)
```

Comment:
I suppose here we can use a capital letter

Reply:
fixed